integration-tests/README.md

```diff
@@ -78,7 +78,14 @@ Basic Use Cases described above are verified in all the domain configurations. A
 | Two Operators using same target domains namespace | create chart using target domains namespace as already running first operator, verify that deployment fails with expected error |
 | Operator Helm Chart using not preexisted target domains namespace | create chart using not preexisted target domains namespace as already running first operator, verify that deployment fails with expected error |
 
+Situational Configuration Override Usecases
```

Comment:
For each set of tests, can you indicate whether it verifies whether the override functionally took effect?  (E.g. if changing a DB username ws tested using a running DB.)   

Does this write-up cover the tests that other folks on the QA team wrote, or the ones Juan H & John G wrote, or just the ones you wrote?

Reply:
The domain config and JDBC resource tests verifies the override functionality took effect.
For the WLDF and JMS the verification stops with checking with the serverConfig MBean tree for expected config changes.

Reply:
Please add this information to each test (e.g. 'config check via WLST only' or 'checked via WLST and functionally via database connections).    

Also please list all known sit cfg tests (those from Xian, Huiling, Davis, yours, John G, Juan H, Vanaja, Pani, ...?).    Or if this check-in doesn't include them, then please work with these folks to enumerate them and work out a way to make sure they all get listed here eventually...  Thanks!

Reply:
Updated the README. This check in doesn't include the tests by others. The only other test that is in functional integration tests is the one in ITOperator which is already documented in the README.